config/hof-defaults.js

```diff
@@ -22,7 +22,20 @@ const defaults = {
   env: process.env.NODE_ENV || 'development',
   gaTagId: process.env.GA_TAG || 'Test-GA-Tag',
   ga4TagId: process.env.GA_4_TAG,
-  gtmTagId: process.env.GTM_TAG || false,
+  // added to allow support for multiple HOF forms using GTM to customize how they track page views
+  gtm: {
+    tagId: process.env.GTM_TAG || false,
+    config: {},
+    composePageName: function (page, convertPage, serviceName) {
+      switch (serviceName) {
+        case 'ETA':
+          return 'ETA | Customer Contact | ' + convertPage(page);
+        // Add other services here...
+        default:
+          return convertPage(page);
+      }
+    }
+  },
```

Comment:
We want to avoid any mention of any service in HOF.  I would just pass it in a config file for ETA

Reply:
Why would you need a switch function? It wouldn’t it just execute one function on a servicr